rulesets/src/collections.ruleset.yml

```diff
@@ -1,31 +1,33 @@
 rules:
   ##### General #####
-  sps-no-collection-paging-capability:
-    description: "Response bodies from collection endpoints SHOULD offer paging capability."
-    severity: warn
-    given: $.paths[?(!@property.match(/.*\/\{[^}]+\}$/))].get.responses['200'].content.application/json.schema.properties
-    then:
-      - field: "paging"
-        function: truthy
-      - field: "paging.type"
-        function: pattern
-        functionOptions:
-          match: "object"
+  # See https://atlassian.spscommerce.com/browse/DPE-286
+  # sps-no-collection-paging-capability:
+  #   description: "Response bodies from collection endpoints SHOULD offer paging capability."
+  #   severity: warn
+  #   given: $.paths[?(!@property.match(/.*\/\{[^}]+\}$/))].get.responses['200'].content.application/json.schema.properties
+  #   then:
+  #     - field: "paging"
+  #       function: truthy
+  #     - field: "paging.type"
+  #       function: pattern
+  #       functionOptions:
+  #         match: "object"
 
   ##### Root Element #####
-  sps-collection-missing-results-array:
-    description: "Response bodies must have a root element called results and is an array of objects."
-    severity: error
-    given: $.paths[?(!@property.match(/.*\/\{[^}]+\}$/))].get.responses['200'].content.application/json.schema.properties.results
-    then:
-      - field: type
-        function: pattern
-        functionOptions:
-          match: "array"
-      - field: items.type
-        function: pattern
-        functionOptions:
-          match: "object"
+  # https://atlassian.spscommerce.com/browse/DPE-286
+  # sps-collection-missing-results-array:
+  #   description: "Response bodies must have a root element called results and is an array of objects."
+  #   severity: error
+  #   given: $.paths[?(!@property.match(/.*\/\{[^}]+\}$/))].get.responses['200'].content.application/json.schema.properties.results
```

Comment:
Can we keep this one since it expects the property of results in the root?

Reply:
Re-enabled!